image/image.go

```diff
@@ -275,6 +275,11 @@ func snapChannel(name string, model *asserts.Model, opts *Options, local *localI
 		// fallback to default channel
 		snapChannel = opts.Channel
 	}
+	if snapChannel != "" {
+		if _, err := channel.ParseVerbatim(snapChannel, "_"); err != nil {
```

Comment:
We sometimes use `"-"` or `"_"` for arch to avoid the default one. Maybe as a drive-by, it'd be useful to add a const in the `snap/channel` package? eg.
```
const NoDefaultArchitecture = "-"
```